plugins/extractors/bigtable/bigtable.go

```diff
@@ -114,9 +126,9 @@ func (e *Extractor) getTablesInfo(ctx context.Context, instances []string, proje
 					return
 				}
 				familyInfoBytes, _ := json.Marshal(tableInfo.FamilyInfos)
-				results = append(results, assets.Table{
+				emit(models.NewRecord(&assets.Table{
```

Comment:
I thought we were against this before? Using pointer to pass data instead of struct? What made us change?

Reply:
in most cases, I want it to be a struct instead of pointer.

But since we are using proto as the model, the linter points out that we are not supposed to pass proto by value because it will copy `sync.Mutex` from `protoimpl.MessageState`.

Since it is almost like a rule in using proto then I guess we kind of have to use pointer instead of value.

Reply:
Nice, also this will also remove the need for reflection hack from our sinks, right?

Reply:
created an issue https://github.com/odpf/meteor/issues/216 to fix kafka sink (where we did the reflection hack).
There was another problem and planning to fix it along with removing the hack there.